Symfony/CS/ConfigurationResolver.php

```diff
@@ -84,6 +99,10 @@ public function getFixers()
 
     public function getProgress()
     {
+        if (!method_exists($this->config, 'getHideProgress')) {
+            return $this->options['progress'];
```

Comment:
Please, add a TODO like
`// TODO: following condition should be removed on 2.0 line and method should be added to ConfigInterface`